rules/downgrade-php72/tests/Rector/FunctionLike/DowngradeReturnObjectTypeDeclarationRector/DowngradeReturnObjectTypeDeclarationRectorTest.php

```diff
@@ -33,6 +33,6 @@ protected function getRectorClass(): string
 
     protected function getPhpVersion(): int
     {
-        return PhpVersionFeature::OBJECT_TYPE;
+        return PhpVersionFeature::OBJECT_TYPE - 1;
```

Comment:
This is not needed. Mniminal version to parse `object` type is PHP 7.2.

Reply:
The PHP 7.2 version is already requried above:

![image](https://user-images.githubusercontent.com/924196/101480433-fb881380-3953-11eb-8ad8-2248e411bff5.png)

Reply:
removed.